woocommerce-services.php

```diff
@@ -1372,12 +1372,12 @@ function add_plugin_action_links( $links ) {
 		}
 
 		function enqueue_wc_connect_script( $root_view, $extra_args = array() ) {
-			$wcs_connection_schemas = $this->api_client->get_service_schemas();
+			$is_alive = $this->api_client->is_alive_cached();
```

Comment:
I think we actually want to use auth_test() here and hit the connection test so `wcs_server_connection` will be true if the server is up and the site is connected to jetpack. We should still make a cached version of `auth_test()` and use it but be sure to invalidate the cache when jetpack is connected or disconnected.

Reply:
I think that this is already taken care of.
The script is enqueued only if Jetpack is connected or in dev mode. I think that this is enough.

Reply:
https://github.com/Automattic/woocommerce-services/blob/develop/woocommerce-services.php#L437

Reply:
Me too